client/object_put.go

```diff
@@ -58,10 +58,15 @@ func (x ResObjectPut) StoredObjectID() oid.ID {
 }
 
 // ObjectWriter is designed to write one object to NeoFS system.
+type ObjectWriter interface {
+	io.WriteCloser
+	GetResult() ResObjectPut
+}
+
+// SingleObjectWriter implements [ObjectWriter].
 //
-// Must be initialized using Client.ObjectPutInit, any other
-// usage is unsafe.
-type ObjectWriter struct {
+// Must be initialized using [Client.ObjectPutInit], any other usage is unsafe.
+type SingleObjectWriter struct {
```

Comment:
BTW, it's not exactly `single`. It can be sliced server-side. `default` maybe.

Reply:
The `default` sounds ok, renamed